riscv/src/runtime.rs

```diff
@@ -281,17 +281,17 @@ impl Runtime {
             "std::machines::hash::keccakf::KeccakF",
             None,
             "keccakf",
+            vec![],
             [format!(
-                "instr keccakf {} ~ keccakf.keccakf {}, {}, STEP ->;", // instr keccakf X, Y -> ~ keccakf X, Y, STEP ->; (sub calls asm)
-                instr_register_params(0, 2, 0), // one pointer for input and one pointer for output (both as input arguments)
+                "instr keccakf link ~> keccakf.keccakf({}, {}, STEP);",
```

Comment:
I think this needs to be changed again because of registers in memory. I think instruction `keccakf` should now receive register ids via assignment registers and then use links to load memory addresses from those addresses. The `poseidon` instruction works that way now, you can check what it does, it should be the same I think.

Reply:
Done